implementation/test/BTCETHPriceFeedTest.js

```diff
@@ -37,7 +36,7 @@ contract('BTCETHPriceFeed', (accounts) => {
       // 7152.55 / 142.28 = 50.2709446162
       // 50.2709446162 * 10^10
       // 502,709,446,162 wei
-      expect(price).to.eq.BN('502709446162')
+      expect(price).to.be.bignumber.equal(new BN('502709446162'))
```

Comment:
Do we gain anything from using this (much more verbose) version vs continuing to use `bn-chai`?

Reply:
we gain code size.
going back to `to.eq.BN`